### DISCUSSION TITLE
Custom Client for iOS

### DISCUSSION THREAD

**Discussion by DaliborS89** [rustdesk/rustdesk-server-pro#865] [Q&A] [answered]
Hello everyone,
I have question, you can create RustDesk for iOS for Custom Clients? Beacuse I can create only Client for Win, MacOS, Android and Rasberry. I know we can download in Appstore RustDesk Client but not good communication with self-server. I can not connected in mobile.
You could me help with issue?
![image](https://github.com/user-attachments/assets/c9f37995-a1f0-4a8a-bd4c-e9c7b03daaeb)

**Comment by DaliborS89** [comment]
Hello, everyone,
I need information for our IT team and my management. I need information about the Rust Desk client for iOS mobile devices and if I can create my own client? Can you please write me the results of the solution for is the implementation possible on a custom server?

**Reply by rustdesk** [reply, marked as answer]
Impossible, because it is impossbile to generate a standalone apk file for you to install on iOS like android. We have no way to upload all custom clients to apple store.

**Reply by brainway** [reply]
is it possible through MDM?

**Reply by khushaal-nandwani** [reply]
> Impossible, because it is impossbile to generate a standalone apk file for you to install on iOS like android. We have no way to upload all custom clients to apple store.

Maybe we can make a build (for ios) where on login it fetches the client details (icon, settings etc) based on what acccount is that linked to?
